package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "elasticio-sailor-nodejs",
   "description": "The official elastic.io library for bootstrapping and executing for Node.js connectors",
-  "version": "2.2.2",
+  "version": "2.2.3.dev",
```

should be `dev.1` not `dev`
```suggestion
  "version": "2.2.3.dev.1",
```